### PR TITLE
Wool Pickup Tracking

### PR DESCRIPTION
**Pull request by AtditC**
Tracks wool pickups and gives 3 XP per wool pick up.

### CODE REVIEW

**Comment by jellz** [comment]
This PR has conflicts preventing it from being merged

**Comment by AtditC** [comment]
Fixed it

File: models/minecraft.js [hunk 1]
```diff
@@ -66,16 +66,18 @@ var MinecraftUser = new Schema({
...
-    obj.xp = ((obj.wins ? obj.wins : 0) * 10) + ((obj.losses ? obj.losses : 0) * 5) + ((obj.wool_destroys ? obj.wool_destroys : 0) * 3) + ((obj.wool_placements ? obj.wool_placements : 0) * 12) + ((obj.kills ? obj.kills : 0) * 2);
+    obj.level = parseInt(0.6 * Math.sqrt(((obj.wins ? obj.wins : 0) * 10) + ((obj.losses ? obj.losses : 0) * 5) + ((obj.wool_destroys ? obj.wool_destroys : 0) * 3) + ((obj.wool_pickups ? obj.wool_pickups : 0) * 3) + ((obj.wool_placements ? obj.wool_placements : 0) * 12) + ((obj.kills ? obj.kills : 0) * 2)), 10) + 1
+    obj.levelRaw = 0.6 * Math.sqrt(((obj.wins ? obj.wins : 0) * 10) + ((obj.losses ? obj.losses : 0) * 5) + ((obj.wool_destroys ? obj.wool_destroys : 0) * 3) + ((obj.wool_pickups ? obj.wool_pickups : 0) * 3) + ((obj.wool_placements ? obj.wool_placements : 0) * 12) + ((obj.kills ? obj.kills : 0) * 2)) + 1
+    obj.xp = ((obj.wins ? obj.wins : 0) * 10) + ((obj.losses ? obj.losses : 0) * 5) + ((obj.wool_destroys ? obj.wool_destroys : 0) * 3) + ((obj.wool_pickups ? obj.wool_pickups : 0) * 3) + ((obj.wool_placements ? obj.wool_placements : 0) * 12) + ((obj.kills ? obj.kills : 0) * 2);
```
Comment (lukechatton) [inline models/minecraft.js]: What in the world am I reading??? 😨

Comment (Draem) [reply models/minecraft.js]: whAt hte fuck is that

Comment (jellz) [reply models/minecraft.js]: It's the algorithm for XP and level

Comment (Draem) [reply models/minecraft.js]: Imagine having unreadable code like that in production, yikes

Comment (jellz) [reply models/minecraft.js]: Yeah I was confused when I first read the code for this PR

Comment (Draem) [reply models/minecraft.js]: :refactorthis: 🎉

**Pull request merged**